lib/games_puzzles_algorithms/games/hex/game_state.py

```diff
@@ -201,6 +201,13 @@ def root(self, *dimensions):
             WinDetector.root(NUM_PLAYERS)
         )
 
+    def reset(self):
+        """Reset the board."""
+        player = color_to_player(COLORS["white"])
+        board = self.clean_board(*self.board.size())
+        win_detector = WinDetector.root(NUM_PLAYERS)
+        self.__init__(player, board, self.win_detector)
```

Comment:
Oh, that's neat, I didn't think about reseting an object by calling `__init__` from inside a method, thanks.